source/settings/plugin/fts-dovecot-plugin.rst

```diff
@@ -100,15 +86,11 @@ Example configurations for different object storage backends:
 * :ref:`s3_example_configuration`
 
 
-.. _plugin-fts-dovecot-setting-fts_dovecot_prefix:
-
-``fts_dovecot_prefix``
-----------------------
-
-.. versionadded:: v2.3.5
-
-- Default: ``no``
-- Values:  :ref:`string`
+.. dovecot_plugin:setting:: fts_dovecot_prefix
+   :added: v2.3.5
+   :default: no
+   :plugin: fts-dovecot
```

Comment:
Multiple responses here :)

1. This setting isn't about config.  It's about the actual plugin name displayed in documentation.  See https://doc.dovecot.org/settings/plugin/ - this is how the plugin is currently displayed to user ("fts-dovecot", not "fts_dovecot").
2. This is irrelevant anyway, since this is just populating a data field in python (currently, it's doing nothing more than appending the plugin identifier to an anchor link).  If we wanted to auto-generate some config in the future, we can just do via a simple replace/translation if we needed to.